### ISSUE TITLE
Import FOB  Files for BC 14

### ISSUE THREAD

**Issue by duichwer** [kine/navertical_tasks#8]
Is there a way to import FOB files for Business Central 14 before compiling the BaseApp?

**Comment by kine** [comment]
No, the tasks are focusing to AL only. I am not working with hybrid. But you can do that by other tools like powershell
